package.json

```diff
@@ -25,7 +25,9 @@
     "bootstrap": "^4.1.3",
     "core-js": "^2.5.4",
     "rxjs": "^6.0.0",
-    "zone.js": "~0.8.26"
+    "zone.js": "~0.8.26",
+    "@auth0/angular-jwt": "^2.0.0",
+    "jwt-decode": "^2.2.0"
```

Comment:
are you using this anywhere?

Reply:
Can be implemented without jwt and entirely by using observables, please try that. Thanks